multiapps-controller-core/src/test/java/org/cloudfoundry/multiapps/controller/core/parser/StagingParametersParserTest.java

```diff
@@ -0,0 +1,74 @@
+package org.cloudfoundry.multiapps.controller.core.parser;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import org.cloudfoundry.multiapps.common.SLException;
+import org.cloudfoundry.multiapps.controller.core.model.SupportedParameters;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class StagingParametersParserTest {
+
+    private StagingParametersParser parser;
+    private List<Map<String, Object>> parametersList;
+
+    @BeforeEach
+    void setup() {
+        parser = new StagingParametersParser();
+        parametersList = new ArrayList<>();
+    }
+
+    @Test
+    void testValidateLifecycleWithCnbAndValidBuildpacks() {
+        parametersList.add(Collections.singletonMap("lifecycle", "cnb"));
+        parametersList.add(Collections.singletonMap("buildpacks", List.of("custom-buildpack-url")));
+        assertDoesNotThrow(() -> parser.parse(parametersList));
```

Comment:
Isn't it better to validate fields of parsed object that includes proper values?